node/src/types/timestamp.rs

```diff
@@ -123,7 +123,8 @@ impl AddAssign<TimeDiff> for Timestamp {
     }
 }
 
-impl Sub<TimeDiff> for Timestamp {
+#[cfg(test)]
+impl std::ops::Sub<TimeDiff> for Timestamp {
```

Comment:
Common coding style would be to `#[cfg(test)] use std::ops::Sub`, but if @Fraser999 does not catch you, you're fine :)

Reply:
Too late! :stuck_out_tongue: